src/bundle/Controller/ContentTypeController.php

```diff
@@ -375,8 +396,10 @@ public function editAction(Request $request, ContentTypeGroup $group, ContentTyp
             null,
             ['contentType' => $contentType]
         );
-        $form->handleRequest($request);
 
+        $this->addMetaFieldDefinitions($contentTypeDraft);
```

Comment:
tbh, as together with that method there are 4 new one added, I am not sure this responsibility still belongs to Controller, and maybe it should be extracted to some separate service with unit tests.